pet-nutrition-service/eureka-client.js

```diff
@@ -0,0 +1,45 @@
+const logger = require('pino')();
+const axios = require('axios');
+const ip = require('ip');
+
+const URL = process.env.EUREKA_SERVER_URL || `http://localhost:8761/eureka`;
+
+/**
+ * Register with Eureka server.
+ */
+
+module.exports = function (app, port) {
+  axios.post(`${URL}/apps/${app}/`, {
+    instance: {
+      hostName: ip.address(),
+      instanceId: `${app}-${port}`,
+      vipAddress: `${app}`,
+      app: `${app.toUpperCase()}`,
+      ipAddr: ip.address(),
+      status: `UP`,
+      port: {
+          $: port,
+          "@enabled": true
+      },
+      dataCenterInfo: {
+          "@class": `com.netflix.appinfo.InstanceInfo$DefaultDataCenterInfo`,
```

Comment:
What does this do? Anything related to Netflix?

Reply:
It's just part of the the client spec. The other languages do it, it's just explicit here since I wrote my own client. See https://github.com/search?q=repo%3ANetflix%2Feureka%20com.netflix.appinfo.InstanceInfo%24DefaultDataCenterInfo&type=code

Reply:
I believe when you do this, it relies on the data you provide versus pulling metdata from a different data center provider.